src/common/contentPreference.ts

```diff
@@ -516,3 +518,28 @@ export const unblockEntity = async ({
   // currently unblock is just like unfollow, eg. remove everything from db
   return unfollowEntity({ ctx, id, entity, feedId });
 };
+
+export const whereNotUserBlocked = (
+  qb: QueryBuilder,
+  {
+    userId,
+    feedId,
+  }: Partial<{
+    userId: string;
+    feedId: string;
+  }>,
+) => {
+  const feedIds = feedId ? [feedId, userId] : [userId];
```

Comment:
userId can be optional as well no, so basically if nothing was passed this query would query all the blocked user rows, we need to guard from function signature and inside the query

Reply:
`userId` is mandatory, i fixed this! ✔️